smoke-tests/apps/JmxMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JmxMetricTest.java

```diff
@@ -140,7 +137,7 @@ private void verifyJmxMetricsSentToOtlpEndpoint() {
 
   private void verifyJmxMetricsSentToBreeze() throws Exception {
     List<Envelope> metricItems =
-        testing.mockedIngestion.waitForItems(JmxMetricTest::isJmxMetric, 1, 10, TimeUnit.SECONDS);
+        testing.mockedIngestion.waitForItems(JmxMetricTest::isJmxMetric, 7, 10, TimeUnit.SECONDS);
```

Comment:
why is 7? You added 12 new jvm metric names.

Reply:
This is a good point, let me recount to make sure I've got the right # here

Reply:
I took another look and this method filters on jmx metrics only (not the otlp ones). So think 6 jmx metrics for all java versions + at least 1 optional GC metric.